### DISCUSSION TITLE
Tenda w311m not working

### DISCUSSION THREAD

**Discussion by mhkanon** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#66] [General]
I have a "Tenda w311m" device it not working big sur 11.4. Tested

**Comment by chris1111** [comment]
https://github.com/chris1111/Wireless-USB-OC-Big-Sur-Adapter/discussions/62
Works  here

**Reply by mhkanon** [reply]
Look that is "w311mi(mini)" and my one is "w311m". They are two different devices.
